frontend/mobile/lib/pages/library.dart

```diff
@@ -42,21 +43,30 @@ class _LibraryState extends State<Library> {
                       .error(snapshot.error.toString())),
                 );
               } else {
-                workflows = snapshot.data!.data!;
-                return ListView.builder(
-                  itemCount: workflows.length,
-                  itemBuilder: (context, index) {
-                    return WorkflowTile(
-                      workflow: workflows[index],
-                      onUpdate: () async {
-                        final newWorkflows = await services.workflows.getAll();
-                        setState(() {
-                          workflows = newWorkflows.data!;
-                        });
+                if (snapshot.hasData) {
+                  if (snapshot.data!.data != null) {
+                    workflows = snapshot.data!.data!;
+                    return ListView.builder(
+                      itemCount: workflows.length,
+                      itemBuilder: (context, index) {
+                        return WorkflowTile(
+                          workflow: workflows[index],
+                          onUpdate: () async {
+                            final newWorkflows =
+                                await services.workflows.getAll();
+                            setState(() {
+                              workflows = newWorkflows.data!;
+                            });
+                          },
+                        );
                       },
                     );
-                  },
-                );
+                  } else {
+                    return const Center(child: Text("No datas available 1"));
+                  }
+                } else {
+                  return const Center(child: Text("No datas available 2"));
+                }
```

Comment:
Are these empty states ? If yes then add them in the translation files instead of hardcoding english (and don't use "1" or "2")